dds/DCPS/transport/framework/TransportSendStrategy.cpp

```diff
@@ -1065,7 +1065,7 @@ TransportSendStrategy::send(TransportQueueElement* element, bool relink)
         if (max_message_size) { // fragmentation enabled
           const size_t avail = this->space_available();
           if (element_length > avail) {
-            VDBG_LVL((LM_TRACE, "(%P|%t) DBG:   Fragmenting\n"), 0);
+            VDBG_LVL((LM_TRACE, "(%P|%t) DBG:   Fragmenting %u > %u\n", element_length, avail), 0);
```

Comment:
`%B` is used for `size_t` logging.  It does seem to be needed so that the `va_arg` is 64-bit